docs/gyakorlat/rest/index.md

```diff
@@ -98,99 +96,90 @@ Készítsünk egy új Web API controllert, ami visszaad egy üdvözlő szöveget
     Foglaljuk össze, mi kell ahhoz, hogy egy WebAPI végpontot készítsünk:
 
     - Leszármazni a `ControllerBase`-ből és az `[ApiController]` attribútumot rátenni az osztályra.
```

Comment:
Írja a szöveg, hogy "Leszármazni a `ControllerBase`-ből". Ugyanakkor a frissített megoldásban nem szerepel a ControllerBase ős. Igaz, így is működik, de mivel mindenhol azt mondjuk hogy erre szükség van (előadáson, itt a gyakorlat felvezetőben), lehet jobb, ha visszatesszük (általában úgyis valóban szükség van rá). Attől még, ha visszakerül, a visszatérés típusa maradhat string ebben a bevezető példában (nem kell ActionResult) , ha úgy szimpatikusabb.

Reply:
jogos, egyértelműsítettem és beraktam a példakódba